tests/framework/PostProcessors/LimitSurface/test_LimitSurface_with_err_bounds.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="all">
+  <TestInfo>
+    <name>framework/PostProcessors/LimitSurface.testLimitSurfaceIntegralPPWihtBoundingError</name>
+    <author>alfoa</author>
+    <created>2020-04-15</created>
+    <classesTested>Models.PostProcessors.LimitSurface, Models.PostProcessors.LimitSurfaceIntegral</classesTested>
+    <description>
+       This test is aimed to check the capability of RAVEN to compute the integral of the Limit Surface (e.g.
+       Failure probability) reporting the bounding error as well (the maximum error, in probability, in the computation of the limit surface )
+    </description>
+    <revisions>
+      <revision author="alfoa" date="2020-04-15">Added test for computation of the error bound</revision>
+    </revisions>
+    <requirements>R-RA-7</requirements>
```

Comment:
Do you need to update the requirements_list.xml?

Reply:
you are right. I added a new requirement.